circle.yml

```diff
@@ -27,7 +27,11 @@ defaults:
     run:
       name: "Build"
       working_directory: ~/build
-      command: cmake --build . --target package -- -j $BUILD_PARALLEL_JOBS
+      command: |
+        cmake --build . --target package -- -j $BUILD_PARALLEL_JOBS
+        mkdir -p ../package
+        . aleth/buildinfo.sh
+        mv aleth.tar.gz ../package/$PROJECT_NAME-$PROJECT_VERSION-$SYSTEM_NAME-$SYSTEM_PROCESSOR.tar.gz
```

Comment:
Can/should this include the commit hash?

Reply:
The commit hash is included in `$PROJECT_VERSION`.